.github/workflows/ci.yaml

```diff
@@ -99,6 +103,17 @@ jobs:
       run: ./Performance/allocations/test-allocation-counts.sh
       env: ${{ matrix.env }}
       timeout-minutes: 20
+    - name: Install jemalloc for benchmarking
+      run: apt update && apt-get install -y libjemalloc-dev
+      timeout-minutes: 20
+    - name: Build Benchmarks
+      working-directory: ./Performance/Benchmarks
+      run: swift build
+      timeout-minutes: 20
```

Comment:
I'm not familiar with the benchmark package, is this step actually necessary? If so should it be built with `-c release`?

Reply:
The `swift package benchmark` will drive the compilation of the benchmark targets and build them in release mode automatically. `swift build` can and should actually be skipped.